style.css

```diff
@@ -20,6 +20,183 @@ body {
   overflow-x: hidden;
 }
 
+/* ----------------------------------------------POP UP display ----------------------------------------------------------- */
+.popup-container {
+  display: none;
+  position: absolute;
```

Comment:
- As requested by the previous reviewer [here](https://github.com/ridabensalem/ridabensalem.github.io/pull/3#discussion_r1062957431) please make sure your popup looks like the Figma design, here are some hints:
  - Let's use `position: fixed` instead of `absolute`
  - and give the popup overlay a blurred background as per the design.